.github/workflows/ci.yaml

```diff
@@ -48,15 +51,33 @@ jobs:
         name: codecov-umbrella # optional
         fail_ci_if_error: true # optional (default = false)
 
-    - name: Docker build
-      if: github.event_name == 'pull_request'
+    - name: Docker build # on PRs. on pushes to forks
+      if: github.event_name == 'pull_request' || (github.repository != 'keikoproj/upgrade-manager' && github.event_name == 'push')
       run: make docker-build
     
-    - name: Build and push Docker images
+    - name: Build and push Docker image with tag master # only on pushes to keikoproj/upgrade-manager
+      if: github.event_name == 'push' && github.repository == 'keikoproj/upgrade-manager'
```

Comment:
Will this trigger if we push to a branch different than master?

Reply:
No, the workflow is defined only for pushes in the master branch:
```
on:
  push:
    branches: 
      - master
```